lib/nfp

```diff
@@ -29,7 +29,7 @@ function namespace_delete {
 function namespace_create {
 
     source $1/openrc neutron service
-    SERVICE_MGMT_NET="l2p_svc_management_ptg"
+    SERVICE_MGMT_NET="l2p_service_management_l2p"
```

Comment:
Why this name is changed ?
The old name is meaningful. The new one has duplicate substring l2p